src/gui/editorwidgets/qgsexternalresourceconfigdlg.cpp

```diff
@@ -253,3 +265,12 @@ void QgsExternalResourceConfigDlg::setRootPathExpression( const QgsProperty &pro
   mRootPathExpression->setText( property.valueAsString( ctx ) );
```

Comment:
@nyalldawson @m-kuhn this was not working in current code (to get the expression in the line edit). It seems `QgsProperty::valueAsString` can't return a value. Any idea?